src/pages/Store/Pickups.vue

```diff
@@ -82,6 +96,7 @@
 import PickupList from '@/components/Pickups/PickupList'
 import KNotice from '@/components/General/KNotice'
 import Markdown from '@/components/Markdown'
+import browser from 'browser-detect'
```

Comment:
Should I rather use [quasar's platform detection](https://quasar-framework.org/quasar-play/android/index.html#/showcase/misc/platform-detection) feature?